### DISCUSSION TITLE
Feature request: Websocket api

### DISCUSSION THREAD

**Discussion by mvanbaak** [aws/serverless-application-model#3500] [Ideas]
**Description:**

This morning GA of the new websocket api in API Gateway was announced.
Would be nice if we can configure this in our SAM templates.

**Comment by bikerp** [comment]
Any idea wheter the Websocket API will be supported?

**Comment by keetonian** [comment]
We prioritize issues by the number of +1's on the issue. I agree that this would be good to support in SAM and that we should implement this feature; I'll try to bump this up in our priorities.

**Comment by sansmoraxz** [comment]
Any movement on this?

Something like `AWS::Serverless::WebSocketApi` where we can directly define the routes, integrations and connectors would help make the workflow far easier to manage.

**Comment by ssenchenko** [comment]
Moving this issue to feature discussions with other features under consideration.
